src/com/esotericsoftware/kryo/util/DefaultGenericsStrategy.java

```diff
@@ -31,9 +30,11 @@
 import java.lang.reflect.TypeVariable;
 import java.util.ArrayList;
 
+import com.esotericsoftware.kryo.Kryo;
+
 /** Stores the generic type arguments and actual classes for type variables in the current location in the object graph.
  * @author Nathan Sweet */
-public class Generics {
+public class DefaultGenericsStrategy implements GenericsStrategy {
```

Comment:
I think this could be moved up to the `kryo` package and made package visible only, to reduce the public api.

Reply:
DOne